modules/multy-s3-bucket/0.2.0/s3-replica-bucket.tf

```diff
@@ -33,13 +33,86 @@ resource "aws_s3_bucket_lifecycle_configuration" "replica" {
   provider   = aws.replicaregion
   depends_on = [aws_s3_bucket_versioning.replica]
   bucket     = aws_s3_bucket.replica.id
+
+  rule {
+    id = "expire_old_vault_backups"
+
+    filter {
+      prefix = "hashicorp-vault-backups/*"
```

Comment:
do we need to wild card it? Example: `**/hashicorp-vault-backups`? or does it just work? similar to my comment below